.github/ISSUE_TEMPLATE/docs_error.yaml

```diff
@@ -0,0 +1,31 @@
+name: API documentation error
+description: Report a typo or error found in API documentation
+labels: ["documentation"]
+body:
+  - type: textarea
+    attributes:
+      label: Description of the issue
+      description: Description of what issue you have discovered. Please include as to why you think the issue is an issue.
```

Comment:
I'd rephrase _Description of what issue you have discovered. Please include as to why you think the issue is an issue._ to _Describe what you discovered and explain why it is an issue._